parsimonious/exceptions.py

```diff
@@ -2,8 +2,11 @@
 
 from parsimonious.utils import StrAndRepr
 
+class ParsimoniousError(Exception):
+    """A base exception class to allow library users to catch any Parsimonious error."""
+    pass
```

Comment:
Small formatting nit: two lines around class definitions matching the rest of the file and pep8.
```suggestion

class ParsimoniousError(Exception):
    """A base exception class to allow library users to catch any Parsimonious error."""
    pass

```